src/components/riskmap/featureCollectionConverter.ts

```diff
@@ -12,7 +12,17 @@ const segmentCoordinates = (data: AccidentData) => {
   return coordinates;
 };
 
-const pointCoordinates = (data: AccidentData) => {
+export const pointCoordinates = (data: AccidentData) => {
+  if (data.Longitude !== undefined && data.Latitude !== undefined) {
+    const log10Long = Math.log10(data.Longitude);
+    const exponentLong = Math.floor(log10Long);
+    const longitude = data.Longitude / Math.pow(10, exponentLong);
```

Comment:
Don't quite understand the algorithm here, but if it converts the coordinates to the correct format, it should be fine.

Reply:
So the JSON parser interprets the lat/long values from RWS (which have that Dutch format with "," in between every 3 digits) as a very long number. So, basically, the JSON parses a coordinate as something like 5134039604351320. 

What I do is I first take the log 10 of the number to see its order of magnitude (in this example it's 15.71). Then, I floor this number (to 15) to get the exponent of the power of 10.  These two steps basically find the power of 10 that's closest to the coordinate value. Finally, I divide the number by 10^exponent (10^15) to format it into a longitude (5.134...).